os_site/src/_includes/_header.njk

```diff
@@ -0,0 +1,36 @@
+<header class= "has-bg-white-off">
+    <nav aria-label="primary" class="c-navbar c-navbar--dark has-bg-black-off">
+        <div class="c-navbar__top">
+            <div class="has-bg-black-off" style="width:70%"></div>
+            <div class="c-navbar__content c-navbar__content--home">
+                <ul class="c-navbar__items t-uppercase">
+                    <li class="c-navbar__item">
+                        <a class="c-navbar__item-content c-navbar__clickable c-navbar__clickable--animated  t-size-xxs" href="/">
+                            <strong>Home</strong>
+                        </a>
+                    </li>
+                    <li class="c-navbar__item">
+                        <a class="c-navbar__item-content c-navbar__clickable c-navbar__clickable--animated  t-size-xxs" href="/about">
+                            <strong>About</strong>
+                        </a>
+                    </li>
+                    <li class="c-navbar__item">
+                        <a class="c-navbar__item-content c-navbar__clickable c-navbar__clickable--animated  t-size-xxs" href="/posts">
+                            <strong>Blog</strong>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <div class="banner">
+        <div class="banner-items has-xl-padding t-align-center has-text-black-off t-lsp-m t-size-b">
+            <div class="logo">
```

Comment:
Just for easier navigation, it might be nice to link the logo back to the homepage for this site.

Reply:
Ah yes, thanks for this! Just added a wrapper to link back to the homepage.

Reply:
perfect!